### PR TITLE
readme: add installation instructions

### PR DESCRIPTION
**Pull request by davidhcefx**
## Description

When I build this project the first time, what I got after running `make` was:
```
npx: command not found
```

So I googled around and installed `nodejs` and other stuff. But then `npm` told me it failed to compile, so I spent more time to found that I have to run `npm i` for the unmet dependencies. So I think maybe a short guidance in the README would be great.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -92,6 +92,15 @@ if (err == HPE_OK) {
 ```
 For more information on API usage, please refer to [src/native/api.h](https://github.com/nodejs/llhttp/blob/main/src/native/api.h).
 
+## Installation
```
Comment (indutny) [inline README.md:95]: Maybe:
```suggestion
## Build Instructions
```

**Review by indutny — CHANGES_REQUESTED** [review]
LGTM, except for one suggestion.

**Review by indutny — APPROVED** [review]
LGTM. Thanks!

**Comment by davidhcefx** [comment]
You’re welcome! Thanks for your approval.

**Pull request merged**